torch/_inductor/ops_handler.py

```diff
@@ -624,13 +636,17 @@ def inner(*args, **kwargs):
             val = getattr(self.parent_handler, name)(*args, **kwargs)
             if name == "indirect_indexing":
                 return val
-            if val not in self.var_names:
-                varname = f"tmp{self.op_count}"
-                self.op_count += 1
-                self.var_names[val] = varname
-                return varname
-            else:
-                return self.var_names[val]
+
+            def count(val):
+                if val not in self.var_names:
+                    varname = f"tmp{self.op_count}"
+                    self.op_count += 1
```

Comment:
A bit weird that frexp counts as two ops but okay.

Reply:
Agreed. Not sure if it's worth fixing that.